modules/ROOT/pages/_partials/component-page-TEMPLATE.adoc

```diff
@@ -0,0 +1,75 @@
+// Filenames follow a syntax component-some-name.adoc
+// So for component-scatter-gather.adocs, the confugration is
+// :component-filename: scatter-gather
+:component-filename: FILENAME_END_HERE
+
```

Comment:
TEMPLATE for component ACB component PAGES that use the partial tags. Stored under _partials only to avoid building the template. Instructions for use to follow in the file, but basically its:

1. copy the file to the /pages directory
2. change the file name to acb-component-some-name.adoc 
3. comment out any includes for optional sections that are unused or sections that can't be completed